python/packages/ai/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "teams-ai"
-version = "1.8.0"
+version = "1.8.1"
```

Comment:
as this is a package bump, we will complete this in a separate PR - please discard this change in the meantime!